docs/serverless/troubleshooting/troubleshoot-endpoints.asciidoc

```diff
@@ -222,3 +222,17 @@ sudo /Library/Elastic/Endpoint/elastic-endpoint test install
 
 If the command output doesn't contain a message about enabling Full Disk Access, the approval was successful.
 =====
+
+[discrete]
+[[disable-self-healing]]
+.Disable {elastic-defend}'s self-healing feature on Windows
+[%collapsible]
+====
+
+{elastic-defend}'s self-healing feature rolls back recent file system changes when a prevention alert is triggered. This feature uses the Windows Volume Snapshot Service. Although it's uncommon for this to cause issues, you can turn off this {elastic-defend} feature if needed.
```

Comment:
```suggestion
{elastic-defend}'s self-healing feature rolls back recent file system and registry changes when a prevention alert is triggered. This feature uses the Windows Volume Snapshot Service. Although it's uncommon for this to cause issues, you can turn off this {elastic-defend} feature if needed.
```

Should we also list known compatibility issues, such as [DFS Replication](https://learn.microsoft.com/en-us/windows-server/storage/dfs-replication/dfsr-overview) and Veeam Backup & Replication?  This repo is public so I can't include put private links here.

Reply:
Hi @gabriellandau, I'm not very familiar with these compatibility issues (I've only recently taken over these docs and feature area) – could you provide some suggested wording you'd like to see included? 🙏

Reply:
Some users have reported compatibility issues between {elastic-defend}'s self-healing feature and filesystem replication features including [DFS Replication](https://learn.microsoft.com/en-us/windows-server/storage/dfs-replication/dfsr-overview) and Veeam Replication.  This may manifest as DFSR Event ID 1102:

> The DFS Replication service has temporarily stopped replication because another application is performing a backup or restore operation. Replication will resume after the backup or restore operation has finished."

There are no known workarounds for this issue other than to disable self-healing.